js.go

```diff
@@ -181,9 +183,29 @@ func (nc *Conn) JetStream(opts ...JSOpt) (JetStreamContext, error) {
 			return nil, err
 		}
 	}
+	return js, nil
+}
+
+// BindJetStream returns a JetStreamContext for messaging and stream management that will NOT create
+// underlying objects like stream or durable. It assumes they exist already.
```

Comment:
Maybe expand a bit on the godoc that is to bind to a JetStream from another account?
```suggestion
// underlying objects like stream or durable on Subscribe. It assumes they exist already and that
// the JetStream context is bound to another account via an import with limited permissions for example.
```